dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ServiceMojos.scala

```diff
@@ -118,7 +122,9 @@ class StartMojo @Inject() (serviceManager: ServiceManager, session: MavenSession
     }
 
     val selectedPort = selectPort(serviceHttpPort, useTls = false)
-    val selectedHttpsPort = selectPort(serviceHttpsPort, useTls = true)
+    val selectedHttpsPort =
+      if (serviceEnableSsl) selectPort(serviceHttpsPort, useTls = true)
+      else -1
```

Comment:
This should also be done further done in `StartExternalProjects`
(not showing in the diff, so commenting here)

Reply:
done (03e05cb53)